File: android/src/main/java/com/wishlist/UIScheduler.java
package com.wishlist;

import com.facebook.proguard.annotations.DoNotStrip;
import com.facebook.react.bridge.UiThreadUtil;

@DoNotStrip
public class UIScheduler {
  @DoNotStrip
  public static void scheduleOnUI(Runnable runnable) {
    UiThreadUtil.runOnUiThread(runnable);
  }
}

File: android/src/main/java/com/wishlist/UIScheduler.kt
package com.wishlist

import com.facebook.proguard.annotations.DoNotStrip
import com.facebook.react.bridge.UiThreadUtil

@DoNotStrip
class UIScheduler {
  companion object {
    @DoNotStrip
    @JvmStatic
    fun scheduleOnUI(runnable: Runnable?) {
      UiThreadUtil.runOnUiThread(runnable)
    }
  }
}
